config/config.go

```diff
@@ -27,6 +27,13 @@ const (
 )
 
 var windowsAbsolutePathPattern = regexp.MustCompile("^[A-Z]{1}:")
+// TODO: add jira issue tracker pattern
+var jiraOriginPattern = regexp.MustCompile(``)
```

Comment:
For Jira, the origin is similar to redmine. It can be any domain (e.g. `https://myjira.com`).

Reply:
resolved in accb53f